src/base/utils/foreignapps.cpp

```diff
@@ -223,9 +231,13 @@ namespace
         // Fallback: Detect python from default locations
         const QFileInfoList dirs = QDir("C:/").entryInfoList({"Python*"}, QDir::Dirs, (QDir::Name | QDir::Reversed));
         for (const QFileInfo &info : dirs) {
-            const QString path {info.absolutePath() + "/python.exe"};
-            if (QFile::exists(path))
-                return path;
+            const QString py3Path {info.absolutePath() + "/python3.exe"};
+            const QString pyPath {info.absolutePath() + "/python.exe"};
+
+            if (QFile::exists(py3Path))
+                return py3Path;
+            else if (QFile::exists(pyPath))
```

Comment:
`else` is redundant.

Reply:
Also it would be better to check each path after its creation:
```c++
const QString py3Path {info.absolutePath() + "/python3.exe"};
if (QFile::exists(py3Path))
    return py3Path;

const QString pyPath {info.absolutePath() + "/python.exe"};
if (QFile::exists(pyPath))
    return pyPath;
```

I would even declare one variable instead of two const but @Chocobo1 will object...

Reply:
> I would even declare one variable instead of two const but @Chocobo1 will object...

I would like to see his response before I change this.

Reply:
Hmm... I won't strongly object, but will suggest. And personally I would choose the code example in https://github.com/qbittorrent/qBittorrent/pull/12301#discussion_r400773247.